src/Domain/EmbeddingMaps/AffineMap.cpp

```diff
@@ -0,0 +1 @@
+
```

Comment:
Following Will's comment elsewhere, I would agree with him that anything we anticipate only be used as a pointer to abstract base class is better off in the cpp file. We need all the compilation speed improvements we can get ;) (my fault, I know)

But to answer your question directly, yes since it's not a class template the static member variable must be initialized inside a cpp file since it will have external linkage.